_releases/2023-01-23-1.7.2-released.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Crystal 1.7.2 is released!
+version: 1.7.2
+summary:
+thumbnail: +
+author:
+---
+
+We are releasing the second patch release of the [1.7](/2023/01/09/1.7.0-released/) series.
+
+Pre-built packages are available on [GitHub Releases](https://github.com/crystal-lang/crystal/releases/tag/1.7.2) and our official distribution channels.
+See [crystal-lang.org/install](https://crystal-lang.org/install/) for installation instructions.
+
+This patch includes two bugfixes for regressions introduced in 1.7.0 and a minor UX bug in the compiler.
+The parser in 1.7.0 added more restrictions on what tokens can follow immediately after the name in a class definition. This was too restrictive and unintentionally affected code that should be considered valid ([#12977](https://github.com/crystal-lang/crystal/pull/12977)).
```

Comment:
Maybe it's my sleepy self, but the way this paragraph is structured was hard to parse for me.

```suggestion

The first regression affected the parser, which in 1.7.0 was modified to restrict the set of tokens that can be placed after the name of a class in its definition. This restriction was however overly strict, and it unintentionally affected valid code ([#12977](https://github.com/crystal-lang/crystal/pull/12977)).
```